auth/basics/js.md

```diff
@@ -87,7 +87,7 @@ you can access the currently logged-in user using [`skygear.currentUser`](https:
 
 ``` javascript
 import skygear from 'skygear';
-import skygearError from 'skygear/lib/error';
+import skygearError from 'skygear';
```

Comment:
I think the change is a step in the right direction, but I also believe it should be `SkygearError` instead of `skygearError` @david90